packages/ui/src/theme/css/component/button.scss

```diff
@@ -8,23 +8,44 @@
   align-items: center;
   border-radius: var(--amplify-components-button-border-radius);
   box-sizing: border-box;
+  font-size: var(--amplify-components-button-font-size);
   font-weight: var(--amplify-components-button-font-weight);
   transition: all var(--amplify-components-button-transition-duration);
-  border-width: var(--amplify-components-button-border-width);
   border-style: var(--amplify-components-button-border-style);
   border-color: var(--amplify-components-button-border-color);
   border-radius: var(--amplify-components-button-border-radius);
+  border-width: var(--amplify-components-button-border-width);
   color: var(--amplify-components-button-color);
-  font-size: var(--amplify-components-button-font-size);
   line-height: var(--amplify-components-button-line-height);
+  user-select: none;
   padding-block-start: var(--amplify-components-button-padding-block-start);
   padding-block-end: var(--amplify-components-button-padding-block-end);
   padding-inline-start: var(--amplify-components-button-padding-inline-start);
   padding-inline-end: var(--amplify-components-button-padding-inline-end);
-  user-select: none;
   // all:unset removes this, so we have to add it back.
   cursor: pointer;
 
+  --amplify-internal-button-disabled-color: var(
```

Comment:
Why are we adding these internal variables? Maybe let's add a comment here for that.